.github/actions/build_moltenvk_dxvk/action.yml

```diff
@@ -55,7 +55,7 @@ runs:
         pushd ${{ inputs.crossover-source }}/dxvk
 
         # disable building the 32bit version
-        sed -i "" "s/build_arch 32/# build_arch 32/g" ./package-release.sh
+        # sed -i "" "s/build_arch 32/# build_arch 32/g" ./package-release.sh
```

Comment:
32Bit DXVK doesn’t work on macOS currently as wine(32on64) doesn’t have vulkan trunks.

Also d3d9 dlls 64Bit dlls can be skipped as those also don’t work due to lacking features of Metal/MoltenVK